### ISSUE TITLE
doesnt work for me:(

### ISSUE THREAD

**Issue by drillaflexa** [3ct0s/discord-keylogger#1]
hi im very new to this and need a little help. whenever i try to run the keylogger is says this "Traceback (most recent call last):
  File "C:/Users/tajis.DESKTOP-A930330/Desktop/keylogger.py", line 1, in <module>
    import keyboard,os
ModuleNotFoundError: No module named 'keyboard'"
Please help me would be much appreciated:)

**Comment by 3ct0s** [comment]
Hey, it appears that you did not execute the pip install -r requirements.txt command. Can you please do that and see if it works?

**Comment by drillaflexa** [comment]
have u got a discord so you can help me there

**Comment by 3ct0s** [comment]
No, I can't really help on discord because of the TOS. Can you please do as I said above?

**Comment by adbed777** [comment]
Same for me Defaulting to user installation because normal site-packages is not writable Requirement alredy satisfied: keyboard ... python 3.9....

**Comment by cdkw2** [comment]
any solution to this yet?
